### PR TITLE
Fixing onbuild 

### PR DESCRIPTION
**Pull request by jhamman**

### CODE REVIEW

File: build.py [hunk 1]
```diff
@@ -58,6 +58,8 @@ def docker_build(image_spec, path, build_args):
     for k, v in build_args.items():
         command += ['--build-arg', f'{k}={v}']
     command.append(path)
+    # debug
+    print(command)
     subprocess.check_call(command, shell=True)
```
Comment (jhamman) [inline build.py]: @yuvipanda - I'm not getting any output from this call and the onbuild image isn't showing up. Any thoughts on what is going on here?

Comment (yuvipanda) [reply build.py]: Any output here? https://github.com/pangeo-data/pangeo-stacks/pull/114/files#diff-2955d5257f635de1df53f55a171ca5c7R139

**Comment by scottyhq** [comment]
Issue: We want to test building the base image and then have all other on-build images run as parallel jobs. Currently the base image builds, but is not among the local docker images for dependent jobs, nor is it currently pushed to a public access point, so we get failures like `manifest for pangeo/base-notebook-onbuild:2020.01.28-3f9989d not found: manifest unknown: manifest
 unknown` 

I'll try a solution which pushes base images to dockerhub with a temporary 'PR' tag modifier. Then when the PR is merged those images can be retagged and the 'PR' tags deleted.

**Comment by scottyhq** [comment]
closing in favor of https://github.com/pangeo-data/pangeo-stacks/pull/118

**Pull request closed without merging**